low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -243,7 +232,11 @@ def validate(self):
         pass
 
     def process(self):
-        Cmd._call_script(f"{self._script_dir}/{self.script}", self._args)
+        sspl_bundle_generate = f"{self._script_dir}/{self.script} {' '.join(self._args)}"
+        output, error, returncode = SimpleProcess(sspl_bundle_generate).run()
+        if returncode != 0:
+            sys.stderr.write("error: %s\n\n" % str(error))
```

Comment:
return exception instead of print and exit.

Reply:
Done.